### PR TITLE
MMCA-4959-Update UI to match requested changes

### PR DESCRIPTION
**Pull request by AdamCoops**

### CODE REVIEW

File: app/views/cash_account_no_transactions.scala.html [hunk 1]
```diff
@@ -38,7 +38,13 @@
 
     @p("cf.cash-account.detail.no-transactions.p1")
 
-    @p("cf.cash-account.top-up.guidance")
+    @link(
+        linkMessageKey = "cf.cash-account.top-up.guidance.text.link",
+        location = "https://www.gov.uk/guidance/paying-into-your-cash-account-for-cds-declarations",
+        linkId = Some("cf.cash-account.top.up.guidance"),
```
Comment (wg-hmrc) [inline app/views/cash_account_no_transactions.scala.html:44]: this does not follow the consistent naming pattern

**Comment by tyagikumarvineet** [comment]
In case of Brand new page, below msg key has been display instead of correct text. Did you test the UI ? Could you please have a look?

cf.cash-account.top-up.guidance

File: app/views/cash_account_v2.scala.html [hunk 2]
```diff
@@ -51,7 +51,11 @@
 
     @viewModel.cashAccountBalance
 
-    @viewModel.cashStatementNotification.map(identity)
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            @viewModel.cashStatementNotification.map(identity)
+        </div>
+    </div
```
Comment (tyagikumarvineet) [inline app/views/cash_account_v2.scala.html:58]: Could you close this tag properly?

File: app/views/cash_account_v2.scala.html [hunk 3]
```diff
@@ -51,7 +51,11 @@
 
     @viewModel.cashAccountBalance
 
-    @viewModel.cashStatementNotification.map(identity)
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```
Comment (tyagikumarvineet) [inline app/views/cash_account_v2.scala.html:55]: Could you please add a test to check this style class as it is has been an important requirement here? If takes time then leave it.

File: app/views/cash_account_no_transactions.scala.html
Comment (tyagikumarvineet) [inline app/views/cash_account_no_transactions.scala.html:1]: Is it in purpose as this page is only being used in old cash account journey?

Comment (tyagikumarvineet) [reply app/views/cash_account_no_transactions.scala.html:1]: Could you reply on this as we only have to make changes in new cash account journey? And why this has been resolved without proper response?

Comment (AdamCoops) [reply app/views/cash_account_no_transactions.scala.html:1]: I've updated both routes which was I assumed your chief concern on this matter? it wasn't clear that you want this work removing. It seems better practice to keep the routes up to date with each other, even if it is the old route. You never know if we might need it again.

File: app/views/cash_account_no_transactions_v2.scala.html [hunk 4]
```diff
@@ -44,9 +44,12 @@
         id = Some("cash-account-authorise-an-agent")
     )
 
-    @p(
-        message = "cf.cash-account.top-up.guidance",
-        id = Some("cash-account-authorise-agent-guidance")
+    @link(
+        linkMessageKey = "cf.cash-account.top-up.guidance.text.link",
+        location = "https://www.gov.uk/guidance/paying-into-your-cash-account-for-cds-declarations",
```
Comment (wg-hmrc) [inline app/views/cash_account_no_transactions_v2.scala.html:49]: Please avoid hard coding urls, lets utilise the `application.conf` and `AppConfig`

File: app/views/cash_account_no_transactions.scala.html
Comment (tyagikumarvineet) [reply app/views/cash_account_no_transactions.scala.html:1]: My concern is that, the changes we should be doing as part of this ticket should only be for new cash account journey. We must not be changing anything related to old cash account journey as this is not the task for this ticket. This page is only being used in old cash account journey hence anything related to old cash account journey must not be changed in this ticket. 

What do you mean by below? Are you aware that the existing(old) cash account journey is still getting used in Production and this must not be impacted due to the new journey till the time we are ready to release new journey? And we must not change anything regarding old journey unless asked. 

"It seems better practice to keep the routes up to date with each other, even if it is the old route"

File: test/views/CashAccountNoTransactionsSpec.scala [hunk 5]
```diff
@@ -33,28 +33,48 @@ class CashAccountNoTransactionsSpec extends SpecBase {
...
-      val model: CashAccountViewModel = CashAccountViewModel(eori, cashAccount)
-
+    "display correct title" in new Setup {
       val viewDoc: Document = view(model)
-
       shouldContainCorrectTitle(viewDoc)
+    }
+
+    "display correct back link" in new Setup {
+      val viewDoc: Document = view(model)
```
Comment (tyagikumarvineet) [inline test/views/CashAccountNoTransactionsSpec.scala:42]: Could you please remove this from here and from other test cases and move to the trait Setup (and use from there) as we are using the same view (created using the same values) in all the cases?

File: test/views/CashAccountNoTransactionsV2Spec.scala [hunk 6]
```diff
@@ -28,21 +28,50 @@ class CashAccountNoTransactionsV2Spec extends ViewTestHelper {
...
 
-    "display correct contents" in new Setup {
+    "display title" in new Setup {
       val viewDoc: Document = view(model)
-
       titleShouldBeCorrect(viewDoc, "cf.cash-account.detail.title")
+    }
+
+    "display back link" in new Setup {
+      val viewDoc: Document = view(model)
```
Comment (tyagikumarvineet) [inline test/views/CashAccountNoTransactionsV2Spec.scala:37]: Could you please remove this from here and from other test cases and move to the trait Setup (and use from there) as we are using the same view (created using the same values) in all the cases?

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**